QUEST_JOBSTEP.tsv

```diff
@@ -3112,9 +3112,9 @@ QUEST_JOBSTEP_20150803_003111	I don't have guts for it
 QUEST_JOBSTEP_20150803_003112	Received enough balm. Restore the Vakarine Statue in Fedimian and Ausrine Statue in Klaipeda.
 QUEST_JOBSTEP_20150803_003113	The Sorcerer Master wants to hear more from Amjinases. You can meet the Dievdirbys Master in the west forest of Siaulai, the Necromancer Master in Mulus Corridor, and the Chronomancer Master in Stele Road.
 QUEST_JOBSTEP_20150803_003114	You've heard about eternal lives from Amjinases. Go back to the Sorcerer Master.
-QUEST_JOBSTEP_20150803_003115	Obtain the keepstakes which you could feel strong regrets
-QUEST_JOBSTEP_20150803_003116	The Necromancer Master wants to have the keepstake with stronger grudge. Obtain the keepstakes that you can feel strong regrets from Bidentis Shrine.
-QUEST_JOBSTEP_20150803_003117	You've collected the keepstakes with strong regrets. Go back to the Necromancer Master.
+QUEST_JOBSTEP_20150803_003115	Obtain the keepsakes which you could feel strong regrets
```

Comment:
I agree with all you've said!  I'm kind of new to this GitHub thing and I was only looking at the keepstake and not at the sentences as a whole heh.  I'll check them back and fix them up.  Just the typo that really caught my eye.  Also thanks on the $ hint, I had no clue!